src/CountUp.js

```diff
@@ -200,4 +195,95 @@ class CountUp extends Component {
   }
 }
 
+const NO_ELEMENT = -1;
```

Comment:
That's an interesting hack. Let's add a comment explaining what -1 allows to do here.